06-building-a-program-start-to-finish/writing-tests/output/app-spec.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
+/**
+ * Dependencies declared globally
+ */
+const InvertedIndex = require('../app/js/app.js'),
+  invIndexInstance = new InvertedIndex(),
+  badFormat = require('./books/bad-format.json'),
+  empty = require('./books/empty.json'),
+  book = require('./books/books.json'),
+  otherBook = require('./books/other-book.json');
+/**
```

Comment:
Not quite sure.

We expect that variables names be in camelCase. if base filename should exactly match the name of its default export, then the export, in this case, should also be in camelCase.